src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -484,6 +484,7 @@ private void addLookupError(Mapping.Field field, BindingResult<CompositePurchase
 
   private void mapAcquisitionUnits(List<CompletableFuture<?>> futures, Map<Mapping.Field, DataSourceResolver> mappings,
     CompositePurchaseOrder compPo, Document doc) {
+    Optional.ofNullable(mappings.get(Mapping.Field.VENDOR)).ifPresent(field -> field.resolve(doc));
```

Comment:
Mapping of ACQUISITION_UNIT should be based on the results of VENDOR mapping

Reply:
@grabsefx :-We are getting organization first then with organization account we are matching user provided account , if matches we are assigning acqUnits to the ORDER.